src/controllers/user-management/user.controller.ts

```diff
@@ -171,6 +170,12 @@ export class UserController {
             return user;
         } catch (err) {
             AuditLog.fail(ActionType.UPDATE, User.name, req.user.userId, id);
+            if (
+                err instanceof QueryFailedError &&
+                err.message.startsWith("duplicate key value violates unique constraint")
+            ) {
+                throw new BadRequestException(ErrorCodes.EmailAlreadyInUse);
```

Comment:
The Post and Put method should return the same error code. The original task suggests: "Mailadressen er allerede tilknyttet en anden bruger. Angiv en anden."